formatter/images.go

```diff
@@ -0,0 +1,6 @@
+package formatter
+
+//Images ...
+type Images struct {
```

Comment:
If it's only for this, maybe you can move this type into `formatter/formatter.go`?

Reply:
Sure will do